scripts/reborn_webui_v2_live_qa/run_live_qa.py

```
@@ -6643,6 +6683,51 @@ def _channel_name_mentioned(text: str, channel_name: str) -> bool:
+    connected user is not a member of.
+
+    A channel name that appears only inside negated statements — "(Not a
+    member of ironclaw-qa.)", "I'm not in marketing" — is an honest
+    disclaimer and never a membership claim. A name mentioned in any
+    non-negated sentence (e.g. the listed member channels) is a claim.
+    """
+    if not _channel_name_mentioned(reply_text, channel_name):
+        return False
+    for sentence in _reply_sentences(reply_text):
+        if _channel_name_mentioned(sentence, channel_name):
+            if not NON_MEMBERSHIP_NEGATION_RE.search(sentence):
```

Ruff (0.16.1)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)